tests/test/lib/test_sgd.cpp

```diff
@@ -231,105 +231,4 @@ TEST_CASE("Test reading errors from state", "[sgd]")
     double actual = faasm::readRootMeanSquaredError(p.nBatches, p.nTrain);
     REQUIRE(abs(actual - expectedRmse) < 0.0000001);
 }
-
-TEST_CASE("Run SGD smoke test", "[sgd]")
```

Comment:
This was useful to spot errors when developing the experiment, but is no longer necessary